anesthetic/samples.py

```diff
@@ -822,6 +822,11 @@ def stats(self, nsamples=None, beta=None):
         beta : float, array-like, optional
             inverse temperature(s) beta=1/kT. Default self.beta
 
+        norm : Series, :class:`Samples`, optional
+            :meth:`NestedSamples.stats` output used for normalisation.
+            Can be either a Series of mean values or Samples produced with
+            matching `nsamples` and `beta`.
+
```

Comment:
Need to add the additional columns to the docstring, ideally in a way which doesn't add four more cases to the Returns paragraph below

Reply:
done